### PR TITLE
Create loop.bat

### PR DESCRIPTION
**Pull request by SamyR0**
### Description

loop script for windows

### CODE REVIEW

**Comment by DerddyDert** [comment]
Failed Travis?

**Comment by SamyR0** [comment]
That's why i'm using circleci. :D
Did u test it ? ( the file)

**Comment by DerddyDert** [comment]
PHP Parse error:  syntax error, unexpected 'public' (T_PUBLIC) in ./src/pocketmine/Player.php on line 1443
xargs: php: exited with status 255; aborting
Lint scan failed!

**Comment by SamyR0** [comment]
That because of the Player.php , not the loop
http://prntscr.com/fgm44m

**Comment by SamyR0** [comment]
That's *

**Comment by DerddyDert** [comment]
I see

**Comment by DerddyDert** [comment]
I am retesting your commit now @SamyR0

File: loop.bat [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+) else (
+    set PHP_BINARY=php
+)
+if exist PocketMine-MP.phar (
+    set POCKETMINE_FILE=PocketMine-MP.phar
+) else (
+    if exist src\pocketmine\PocketMine.php (
+        set POCKETMINE_FILE=src\pocketmine\PocketMine.php
+    ) else (
+        msg * "Couldn't find a valid PocketMine-MP installation..."
```
Comment (Leo3418) [inline loop.bat]: My suggestion is to keep consistency with the existing `start.cmd`, so you might want to change this line according to [this one](https://github.com/GenisysPro/GenisysPro/blob/master/start.cmd#L24).

Comment (SamyR0) [inline loop.bat]: Something like chose if you want or not the loop in the start.cmd ?
DO-loop= "yes" or "no" ?

Comment (Leo3418) [inline loop.bat]: Close. On GNU/Linux the user can choose to loop using flag `-l`. Are you able to implement it on the Windows script?

Comment (SamyR0) [inline loop.bat]: I will try :D

Comment (DerddyDert) [inline loop.bat]: @SamyR0 please finish this out close it.

**Pull request closed without merging**